packages/auth/test/helpers/integration/helpers.ts

```diff
@@ -76,9 +80,15 @@ export function getTestInstance(requireEmulator = false): Auth {
   return auth;
 }
 
-export async function cleanUpTestInstance(auth: Auth): Promise<void> {
+export async function cleanUpTestInstance(
+  auth: Auth,
+  tests?: string
+): Promise<void> {
   await auth.signOut();
-  await (auth as IntegrationTestAuth).cleanUp();
+
+  if (typeof tests === 'undefined') {
```

Comment:
Why does this "tests" parameter determine whether or not we call `cleanUp()`? (Also, why shouldn't the TOTP tests call `cleanUp()`?)